Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -34,7 +34,7 @@ open class _ChatMessageContentView<ExtraData: ExtraDataTypes>: _View, ThemeProvi
     /// The current layout options of the view.
     /// When this value is set the subviews are instantiated and laid out just once based on
     /// the received options.
-    private var layoutOptions: ChatMessageLayoutOptions!
+    private(set) var layoutOptions: ChatMessageLayoutOptions!
```

Comment:
Don't remember why we decided to keep this private 🤔 and don't see any cons of giving public `read` access for this (but with moving away from force-unwrapped):
```swift
public private(set) var layoutOptions: ChatMessageLayoutOptions?
```